.github/workflows/ci_windows.yml

```diff
@@ -0,0 +1,52 @@
+name: Run Windows CI
+
+on: [push, pull_request]
+
+env:
+  DEPS_BUILD_DIR: "C:/projects/nvim-deps"
+  DEPS_PREFIX: "C:/projects/nvim-deps/usr"
+  # Silence/redirect errors due to missing locking support (via libgcov).
+  # GCOV_ERROR_FILE: "$(TEMP)/libgcov-errors.log"
+
+jobs:
+  test:
+    runs-on: windows-2016
+    strategy:
+      matrix:
+        config: [ MINGW_64-gcov, MINGW_32, MSVC_64, MSVC_32 ]
+    name: Build and test
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: C:\projects\nvim-deps
+          key: ${{ matrix.config }}-${{ hashFiles('third-party\**') }}
+
+      - uses: actions/setup-python@v2
+        with:
+          pyhon-version: '2.7'
+
+      - uses: actions/setup-python@v2
+        with:
+          pyhon-version: '3.5'
+
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Run CI
+        run: powershell ci\build.ps1
+        env:
+          CONFIGURATION: ${{ matrix.config }}
+
+      - name: Upload artifacts
```

Comment:
The CI workflow doesn't need to upload artifacts.  That's part of the release workflow.